pkg/controller/direct/secretmanager/secret_controller.go

```diff
@@ -203,7 +204,14 @@ func (a *Adapter) Create(ctx context.Context, op *directbase.CreateOperation) er
 	external := a.id.String()
 	status.ExternalRef = &external
 	status.Name = created.Name
-	return op.UpdateStatus(ctx, status, nil)
+	if err = op.UpdateStatus(ctx, status, nil); err != nil {
```

Comment:
```suggestion
	if err := op.UpdateStatus(ctx, status, nil); err != nil {
```

Reply:
Let me fix that in follow up PRs. have some other comments and nits want to resolve as well.